src/regex/lib_pcre.cr

```diff
@@ -1,17 +1,25 @@
 @[Link("pcre")]
 lib LibPCRE
+  JIT_ENABLED = begin
+    LibPCRE.config CONFIG_JIT, out enabled
+    1 == enabled
+  end
+
   alias Int = LibC::Int
 
   type Pcre = Void*
   type PcreExtra = Void*
   fun compile = pcre_compile(pattern : UInt8*, options : Int, errptr : UInt8**, erroffset : Int*, tableptr : Void*) : Pcre
+  fun config = pcre_config(what : Int, where : Int*) : Int
   fun exec = pcre_exec(code : Pcre, extra : PcreExtra, subject : UInt8*, length : Int, offset : Int, options : Int, ovector : Int*, ovecsize : Int) : Int
   fun study = pcre_study(code : Pcre, options : Int, errptr : UInt8**) : PcreExtra
   fun free_study = pcre_free_study(extra : PcreExtra) : Void
   fun full_info = pcre_fullinfo(code : Pcre, extra : PcreExtra, what : Int, where : Int*) : Int
   fun get_stringnumber = pcre_get_stringnumber(code : Pcre, string_name : UInt8*) : Int
   fun get_stringtable_entries = pcre_get_stringtable_entries(code : Pcre, name : UInt8*, first : UInt8**, last : UInt8**) : Int
 
+  CONFIG_JIT = 9
```

Comment:
Can we put this before the definition of `JIT_ENABLED`? That way this constants won't be lazily initialized.

Also maybe `JIT_ENABLED` could be a static method instead of a constant... no need to initialize it all the time in every program just for this spec.

Reply:
> Also maybe JIT_ENABLED could be a static method instead of a constant

This is what I originally tried to do, but apparently you can't have methods within a `lib`?

```cr
syntax error in 'STDIN:6:3': unexpected token: def
```

> Can we put this before the definition of JIT_ENABLED? That way this constants won't be lazily initialized.

Sure.

Reply:
My thinking was not wanting to expose the internals of the lib, but given it's an internal spec it would probably fine to just do the logic there. Can always revisit how to expose it publicly later if needed.

Reply:
Thank you!

Also, these were just suggestions, feel free to wait and see what others think about this.